### PR TITLE
Merging the .gitignore files into a single file, and organizing it

### PR DESCRIPTION
**Pull request by Windchild292**
This cleans up the directory a bit.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -3,19 +3,76 @@
 /RemoteSystemsTempFiles/
 /bin/
 /.settings/
+*/.settings/
```
Comment (sixlettervariables) [inline .gitignore]: Does `**/.settings/` not cover them all?

Comment (Windchild292) [reply .gitignore]: It does, pushed that and cleaned up any duplicates.

File: .gitignore [hunk 2]
```diff
@@ -14,18 +14,18 @@ Thumbs.db
...
-**/.idea/
-**/.vscode
-**/.classpath
-**/.project
-**/.classpath
-**/settings_local.gradle
-**/properties_local.gradle
+out/
+.idea/
+.vscode
```
Comment (sixlettervariables) [inline .gitignore]:
```suggestion
.vscode/
```

**Comment by Windchild292** [comment]
Done

**Pull request merged**